src/Sylius/Bundle/ApiBundle/Resources/config/api_resources/Payment.xml

```diff
@@ -22,18 +22,29 @@
 
         <collectionOperations>
             <collectionOperation name="get">
-                <attribute name="order">
-                    <attribute name="position">ASC</attribute>
+                <attribute name="filters">
+                    <attribute>sylius.api.search_payment_filter</attribute>
                 </attribute>
             </collectionOperation>
         </collectionOperations>
 
         <itemOperations>
             <itemOperation name="get" />
+            <itemOperation name="complete_payment">
```

Comment:
I'm wondering if this name should not correspond to the path ending and controller function name